<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>名称</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>加热效率</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="pbTreeviewChg">
         <property name="text">
          <string>更改</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>隔热管内径(mm)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>高温设定(°C)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QDoubleSpinBox" name="dsbHeatRate">
         <property name="minimum">
          <double>60.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="sbHeatLength">
         <property name="minimum">
          <number>1000</number>
         </property>
         <property name="maximum">
          <number>2500</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="pbTreeviewDel">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="leName">
         <property name="text">
          <string>114*76</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="sbHighTemp">
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>加热棒长度(mm)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="pbTreeviewAdd">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>隔热管外径(mm)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="pbUseCur">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QDoubleSpinBox" name="sbInnerDiam">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>40.000000000000000</double>
         </property>
         <property name="maximum">
          <double>101.000000000000000</double>
         </property>
         <property name="value">
          <double>76.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QDoubleSpinBox" name="sbOutDiam">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>73.000000000000000</double>
         </property>
         <property name="maximum">
          <double>139.000000000000000</double>
         </property>
         <property name="value">
          <double>114.299999999999997</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tvType"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionaddItem">
   <property name="text">
    <string>addItem</string>
   </property>
  </action>
  <action name="actiondeleteItem">
   <property name="text">
    <string>deleteItem</string>
   </property>
  </action>
  <action name="actionalterItem">
   <property name="text">
    <string>alterItem</string>
   </property>
  </action>
  <action name="actionuseItem">
   <property name="text">
    <string>useItem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setting</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setting</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
